src/components/Comments/CommentForm.js

```diff
@@ -51,8 +53,10 @@ class CommentForm extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (!nextProps.isLoading) {
-      this.setState({ inputValue: nextProps.inputValue || '' });
+    if (!nextProps.isLoading && nextProps.inputValue !== '') {
```

Comment:
You could do this instead 
```
if ((!nextProps.isLoading && nextProps.inputValue !== '') || nextProps.commentSubmitted) {
```
That would make the code shorter as we dont need the next lines duplicated.